### ISSUE TITLE
Have an empty url parameter return an empty string

### ISSUE THREAD

**Issue by clandmeter** [kernelsauce/turbo#205]
From what i can tell by using turbo, currently if i specify an url parameter like /home?foo turbo will consider this as a not set and will assign the default value if specified. It would however be nice if there is a difference in no argument specified and an empty argument specified.

http://stackoverflow.com/questions/4557387/is-a-url-query-parameter-valid-if-it-has-no-value

**Comment by kernelsauce** [comment]
Yes, I agree. I will consider this for v2.0, as it will break backwards compatibility for existing applications. Thank you for your opinion @clandmeter. )

**Comment by clandmeter** [comment]
I guess now 2.0 is released, this will not be included in the 2.x releases?

**Comment by kernelsauce** [comment]
Hi. Well, it may be included in 2.0.1 if it can be added without breaking compatibility. E.g like adding a argument to allow blanks... I have not had the time to look at this yet. I think its pretty easy to make that kind of a change.

**Comment by clandmeter** [comment]
Nice, looking forward to it.

PS, can you update front page to reflect the current version?
